config/core/manager_arguments.yaml

```diff
@@ -2,4 +2,3 @@
   path: "/spec/template/spec/containers/0/args"
   value:
     - "--enable-leader-election"
```

Comment:
I thought we no longer needed this file, since it only contains the default arguments now?

Reply:
It's already in repo and not necessary to delete right now.

I could delete it but it's not doing any harm as it is.